### PR TITLE
#358 automatically push interscript.js to interscript-js repo

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #358

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -32,3 +32,16 @@ jobs:
...
+      - name: Build Javascript
+        run: bundle exec rake all
+      - name: Update interscript-js repo
+        env:
+          INTERSCRIPT_CLI_REF: ${{ github.ref }}
+        run: |
+          git config --global user.name "interscript-ci"
+          git config --global user.email "interscript-ci@users.noreply.github.com"
+          git clone https://interscript-ci:${{ secrets.INTERSCRIPT_CI_PAT }}@github.com/interscript/interscript-js
+          cp vendor/assets/javascripts/interscript.js interscript-js/interscript.js
```
Comment (ronaldtse) [inline .github/workflows/release.yml:43]: We also need to `cp vendor/assets/maps/*.json` to the right place.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (CAMOBAP) [reply .github/workflows/release.yml:43]: good catch, done

Comment (ronaldtse) [reply .github/workflows/release.yml:43]: Thanks!

**Comment by ronaldtse** [comment]
Thanks @CAMOBAP !

**Pull request merged**